pgbouncer.yaml

```diff
@@ -40,6 +40,11 @@ pipeline:
 
   - uses: autoconf/make-install
 
+  - runs: |
```

Comment:
Comparing to the[ upstream Dockerfile](https://github.com/cloudnative-pg/pgbouncer-containers/blob/main/Dockerfile#L65-L66), i'm not sure these paths are correct? it looks to copy the files in and also append .example onto the ends of the names. It also looks to create two empty files

Reply:
I created a separate for that but I do want to make necesssary changes here at real pgbouncer:

- https://github.com/wolfi-dev/os/pull/22301

Reply:
Thanks for additional info. On closer inspection, while there isn't an official pgbouncer image (there is a bitnami image and ones from others in the community), they do seem to follow the same pattern and expect these to be present. Other package distros have placed these in the same dir